### ISSUE TITLE
JWT middleware seems dead...

### ISSUE THREAD

**Issue by hyacinthus** [ant0ine/go-json-rest#204]
The author no longer maintains it.

**Comment by Hentioe** [comment]
No, it's not dead. Only [go-json-rest-middleware-jwt](https://github.com/StephanDollberg/go-json-rest-middleware-jwt)-dependent [jwt-go](https://github.com/dgrijalva/jwt-go) versions are upgraded to 3.x and are no longer compatible with the 2.x API. But the middleware is still using the 2.x API.
Add in your glide.yaml:
```` yaml
- package: github.com/dgrijalva/jwt-go
  version: v2.7.0
````
Run the command:
```` bash
glide up
````
And then delete the verdor/StephanDollberg/go-json-rest-middleware-jwt/vendor directory.

**Comment by hyacinthus** [comment]
@Hentioe  Yes, thank you,  I've forked it and merge it to use jwt-go 3.x.

**Comment by PhanHaHus** [comment]
Do you have custom middleware before access API, example of author I cant custom , :(

**Comment by hyacinthus** [comment]
@PhanHaHus  I use Echo in my new projects now... It's much better than go-json-rest :)

**Comment by PhanHaHus** [comment]
@hyacinthus yeah, but Im newbie with golang, and I dont know how to struct project with echo, because go-json-rest have more example :(

**Comment by hyacinthus** [comment]
@PhanHaHus Echo has a cookbook too.
